justfile

```
@@ -118,6 +118,14 @@ mdbook:
     fi
     mdbook serve docs --open --port 8321
 
+# Build debian package
+deb-package:
```

i think we may have other packages later, so perhaps this is better?

```suggestion
package-deb:
```